dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/ChatMessage.cs

```diff
@@ -17,20 +17,27 @@ public abstract class ChatMessageBase
     /// <summary>
     /// Content of the message
     /// </summary>
-    public string Content { get; set; }
+    public string Content { get; set; } = string.Empty;
 
     /// <summary>
     /// Dictionary for any additional message properties
     /// </summary>
     public IDictionary<string, string>? AdditionalProperties { get; set; }
 
     /// <summary>
-    /// Creates a new instance of the <see cref="ChatMessageBase"/> class
+    /// Creates a new instance of the <see cref="ChatMessage"/> class
+    /// </summary>
+    public ChatMessage()
+    {
+    }
+
+    /// <summary>
+    /// Creates a new instance of the <see cref="ChatMessage"/> class
     /// </summary>
     /// <param name="role">Role of the author of the message</param>
     /// <param name="content">Content of the message</param>
     /// <param name="additionalProperties">Dictionary for any additional message properties</param>
-    protected ChatMessageBase(AuthorRole role, string content, IDictionary<string, string>? additionalProperties = null)
+    public ChatMessage(AuthorRole role, string content, IDictionary<string, string>? additionalProperties = null)
     {
         this.Role = role;
         this.Content = content;
```

Comment:
You added `= string.Empty` to the property initializer.  If someone passes in a null role / content, what do you want to do?  Should they still end up as empty? Throw?